grails-app/conf/BuildConfig.groovy

```diff
@@ -43,7 +43,7 @@ grails.project.dependency.resolution = {
     plugins {
         runtime ":hibernate:3.6.10.16"
 
-        runtime ":ala-bootstrap2:2.3-SNAPSHOT"
+        runtime ":ala-bootstrap2:2.4-SNAPSHOT"
```

Comment:
Can we use the non-snapshot versions ? is there a change in 2.4-SNAPSHOT that should be released ?